packages/core/content-releases/server/src/services/release.ts

```diff
@@ -270,14 +270,11 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
 
       return {
         ...action,
-        entry: {
-          id: action.entry.id,
-          contentType: {
-            displayName,
-            mainFieldValue: action.entry[mainField],
-          },
-          locale: action.locale ? allLocalesDictionary[action.locale] : null,
-          status: action.entry.publishedAt ? 'published' : 'draft',
+        locale: action.locale ? allLocalesDictionary[action.locale] : null,
+        contentType: {
+          displayName,
+          mainFieldValue: action.entry[mainField],
+          uid: action.contentType,
```

Comment:
so instead of having the schema and components on the meta you could just store them here and they would come from the same place as mainField and displayName

Reply:
I agree it's a good idea to have a separate function to get this but the reason why I add this to the meta object is to not repeat the same info.

If we put the contentType and components schemas inside the contentType object of each entry it means that if you have 10 entries of X content type, we gonna end with the X's schema 10 times.

Also, components are not specific to one ContentType, they can be used in all the ContentTypes, so I used the same logic: I could return the components related to one ContentType, but if one component is in all the contentTypes I would return it multiple times. So I prefer to return all the components in the same place just one time

Reply:
ah right I see your point 👍